### PR TITLE
EDU-1578 Show rating details

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1578

### CODE REVIEW

File: src/resources/resources.json [hunk 1]
```diff
@@ -3759,26 +3763,26 @@
...
-      "de": "Du hast keine Bewertung für dieses Dokument abgegeben.\n\nBewerte das Dokument, indem du auf die Sterne unten klickst.\n"
+    "stars": {
+      "en": "{starCount, plural, =1 {# star} other {# stars}}",
+      "de": "{starCount, plural, =1 {# Stern} other {# Sterne}}"
     },
-    "existingRatingExplanationMarkdown": {
-      "en": "You have rated this document on {ratingDate} with **{ratingValue, plural, =1 {# star} other {# stars}}**.\n\nYou can change or delete your rating by clicking on the stars or the delete icon below.\n",
-      "de": "Du hast dieses Dokument am {ratingDate} mit **{ratingValue, plural, =1 {# Stern} other {# Sternen}}** bewertet.\n\nDu kannst deine Bewertung ändern oder löschen, indem du auf die Sterne oder das Löschen-Symbol unten klickst.\n"
+    "totalRatingHeader": {
+      "en": "Total rating",
```
Comment (adrianaluchian) [inline src/resources/resources.json:3774]: Should we say "Average rating" instead of "total"? 🤔

Comment (adrianaluchian) [reply src/resources/resources.json:3774]: Or "Global average rating", if you feel like it needs to be more clear that it contains all ratings?

Comment (ahelmberger) [reply src/resources/resources.json]: Changed it to "Average rating" / "Durchschnittliche Bewertung".

File: src/components/rating-dialog.js [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-import React, { useCallback, useEffect, useState } from 'react';
 import { documentRatingShape } from '../ui/default-prop-types.js';
+import React, { Fragment, useCallback, useEffect, useState } from 'react';
 import DocumentRatingApiClient from '../api-clients/document-rating-api-client.js';
 
 const logger = new Logger(import.meta.url);
 
-function RatingDialog({ documentRating, isOpen, onCancel, onOk }) {
-  const { formatDate } = useDateFormat();
+const STAR_STROKE_COLOR = 'rgb(250, 219, 20)';
```
Comment (adrianaluchian) [inline src/components/rating-dialog.js:17]: Just a thought, no changes suggested: I'm split between doing this "the ant way" which is not the educandu way, or working against ant and overwriting the `background-color` of the `ant-progress-bg` CSS class 😅

File: src/resources/resources.json (hunk 1 above)
Comment (adrianaluchian) [reply src/resources/resources.json]: 👍

**Review by adrianaluchian — APPROVED** [review]

File: src/components/rating-dialog.js (hunk 2 above)
Comment (ahelmberger) [reply src/components/rating-dialog.js:17]: I had the same thought, but as the star color is hard-coded in Antd (and therefore we do not have it in CSS anywhere), I went the Antd way here too.

**Pull request merged**